boefjes/boefjes/plugins/kat_rpki/normalize.py

```diff
@@ -10,14 +11,18 @@ def run(input_ooi: dict, raw: bytes) -> Iterable[NormalizerOutput]:
     results = json.loads(raw)
     ooi = Reference.from_str(input_ooi["primary_key"])
 
-    if not results["exists"]:
-        ft = KATFindingType(id="KAT-NO-RPKI")
-        f = Finding(finding_type=ft.reference, ooi=ooi)
-        yield ft
-        yield f
+    address = ip_address(ooi.tokenized.address)
 
-    if not results.get("valid") and not results.get("notexpired"):
-        ft = KATFindingType(id="KAT-EXPIRED-RPKI")
-        f = Finding(finding_type=ft.reference, ooi=ooi)
-        yield ft
-        yield f
+    # if the address is private, we do not need a ROA
+    if address.is_global:
```

Comment:
Should we also avoid running the lookup in the boefje itself when the address is not global? seems like a quick and easy optimization.

Reply:
We absolutely could, just felt this was a more natural split between gathering data in the boefje and drawing conclusions in the normalizer, but I'll move it tomorrow!

Reply:
Optimized the lookup!